gpo/crud.py

```diff
@@ -0,0 +1,38 @@
+"""
+CRUD operations for gpo µservice
+"""
+from sqlalchemy import select
+from sqlalchemy.orm import Session
+
+from . import models, schemas
+
+
+def get_letters(
+    session: Session, skip: int = 0, limit: int = 1000
+) -> list[models.Letter]:
+    """
+    get letters
+    """
+    return (
+        session.execute(select(models.Letter).offset(skip).limit(limit)).scalars().all()
+    )
+
+
+def delete_letters(session: Session, letters: models.Letter):
```

Comment:
Something's amiss with the `letters: models.Letter` type-hint. The hint clearly says that the parameter is a single object of type `models.Letter` but the method is using it as an iterable? But the models.Letter class does not implement the required methods to be used as an iterator from what I can tell in the sqlalchemy docs on `Base` classes.